components/BottomNavigation/examples/BottomNavigationExplicitlySetColorExample.swift

```diff
@@ -0,0 +1,145 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import MaterialComponents
+
+class BottomNavigationExplicitlySetColorExample: UIViewController {
+
+  let appBar = MDCAppBar()
+  var colorScheme = MDCSemanticColorScheme()
+
+  let bottomNavBar = MDCBottomNavigationBar()
+
+  let redButton = MDCButton()
+  let blueButton = MDCButton()
+
+  init() {
+    super.init(nibName: nil, bundle: nil)
+    self.title = "Bottom Navigation Set Color (Swift)"
```

Comment:
I believe this will be set automatically by the breadcrumbs, would you please check?

Reply:
If you don't set the title then the app bar doesn't have a title.